src/js/chartAuthoring.js

```diff
@@ -24,20 +28,75 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                     }
                 }
             },
-            dataEntryPanel: {
-                type: "floe.chartAuthoring.dataEntryPanel",
+            chartAuthoringInterface: {
+                type: "floe.chartAuthoring.templateInjection",
                 createOnEvent: "onTemplatesLoaded",
-                container: "{chartAuthoring}.container",
+                container: "{that}.dom.container",
                 options: {
                     resources: {
-                        template: "{templateLoader}.resources.dataEntryPanel",
-                        dataEntry: "{templateLoader}.resources.dataEntry"
+                        template: "{templateLoader}.resources.chartAuthoring"
+                    },
+                    listeners: {
+                        "onCreate.escalate": "{chartAuthoring}.events.ononChartAuthoringInterfaceReady.fire"
+                    },
+                    components: {
+                        dataEntryPanel: {
+                            type: "floe.chartAuthoring.dataEntryPanel",
+                            createOnEvent: "{chartAuthoring}.events.onPieChartReady",
+                            container: "{chartAuthoring}.dom.dataEntryPanel",
+                            options: {
+                                resources: {
+                                    template: "{templateLoader}.resources.dataEntryPanel",
+                                    dataEntry: "{templateLoader}.resources.dataEntry"
+                                },
+                                modelRelay: {
+                                    source: "{that}.model.dataEntries",
+                                    target: "{floe.chartAuthoring.pieChart}.model.dataSet",
+                                    singleTransform: {
+                                        type: "fluid.transforms.free",
+                                        args: ["{that}.model.dataEntries"],
+                                        func: "floe.chartAuthoring.dataEntriesToPieChartData"
+                                    },
+                                    forward: "liveOnly",
+                                    backward: "never"
```

Comment:
Both ```forward``` and ```backward``` restrictions are unnecessary as,
1. The initial ```{that}.model.dataEntries``` is an empty object;
2.  ```fluid.transforms.free``` doesn't support inverted transformation.